src/main/java/mil/dds/anet/beans/EntityAvatar.java

```diff
@@ -0,0 +1,97 @@
+package mil.dds.anet.beans;
+
+import io.leangen.graphql.annotations.GraphQLInputField;
+import io.leangen.graphql.annotations.GraphQLQuery;
+
+public class EntityAvatar {
+  @GraphQLQuery
+  @GraphQLInputField
+  private String relatedObjectType;
+  @GraphQLQuery
+  @GraphQLInputField
+  private String relatedObjectUuid;
+  @GraphQLQuery
+  @GraphQLInputField
+  private boolean applyCrop;
+  @GraphQLQuery
+  @GraphQLInputField
+  private String attachmentUuid;
+  @GraphQLQuery
+  @GraphQLInputField
+  private Integer cropLeft;
+  @GraphQLQuery
+  @GraphQLInputField
+  private Integer cropTop;
+  @GraphQLQuery
+  @GraphQLInputField
+  private Integer cropWidth;
+  @GraphQLQuery
+  @GraphQLInputField
+  private Integer cropHeight;
+
+  public EntityAvatar() {}
```

Comment:
Is this default constructor needed?

Reply:
A default (parameter-less) constructor without a body is the Java default if you don't declare any (other) constructor, so you can simply remove it. Sonar complains about this, too…